master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -2,41 +2,68 @@
 title: Kubeadm Hosted Install
 ---
 
-This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+This document outlines how to install Calico on a kubeadm cluster.
 If you have already built your cluster with kubeadm, please review the
 [Requirements / Limitations](#requirements--limitations) at the bottom of
 this page. It is likely you will need to recreate your cluster with the
 `--pod-network-cidr` and `--service-cidr` arguments to kubeadm.
 
-Users who have deployed their own etcd cluster outside of kubeadm should
-use the [Calico only manifest](../hosted) instead, as it does not deploy its
-own etcd.
-
 You can easily create a cluster compatible with this manifest by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
 
+## Installation
 
-#### Installation
+The below manifests are compatible with kubeadm clusters initialized with a
+`pod-network-cidr` matching the default pool of `192.168.0.0/16`, as follows:
+
+```shell
+kubeadm init --pod-network-cidr=192.168.0.0/16
+```
+
+### ETCD Datastore
+
+Users who have deployed their own etcd cluster outside of kubeadm should
+use the [Calico only manifest](../hosted) instead, as it does not deploy its
+own etcd.
 
 To install this Calico and a single node etcd on a run the following command
 depending on your kubeadm / kubernetes version:
 
 For Kubeadm stable with Kubernetes version >= v1.6.0:
 
-```
+```shell
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml
 ```
 
 >[Click here to view the above yaml directly.](1.6/calico.yaml)
 
 For Kubeadm 1.5 with Kubernetes version v1.5.x:
 
-```
+```shell
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/1.5/calico.yaml
 ```
 
 >[Click here to view the above yaml directly.](1.5/calico.yaml)
 
+### Kubernetes Datastore
+
+To install Calico with Calico networking, run the following commands to create these necessary RBAC 
+roles and bindings, and launch Calico.
+
+> **Note**: The following RBAC policy is compatible with the Kubernetes v1.7+
```

Comment:
"the Kubernetes v1.7+" --> "Kubernetes v1.7."  

Note the period at the end! :))